tests/ampctl_parse.c

```diff
@@ -1608,7 +1608,7 @@ int set_conf(AMP *my_amp, char *conf_parms)
 
         if (!q)
         {
-            return RIG_EINVAL;
```

Comment:
This change is ok because the code is parsing a line was supposed to contain an equal sign but it is not.